website/static/js/remote/viewModels.js

```diff
@@ -287,17 +287,18 @@ function RoomViewModel() {
   };
 
   self.postPlaylistSort = function(pk, action, target) {
-    target = (typeof target === 'undefined') ? '' : target;
     $('.overlay-playlist').show();
-    var url = '/playlist';
-    url += pk ? '/' + pk : '';
-    url += action ? '/' + action : '';
-    url += target ? '/' + target : '';
+    var data = {};
+    pk ? data.pk = pk : null;
```

Comment:
```js
data.pk = pk || null;
```
Or just 
```js
data.pk = pk;
```
if we don't care about null or unefined.

Here, if pk is falsy, you just declare a null x)
```js
if (pk) {
  data.pk = pk;
}
else {
  null;
}
```